include/vulkan/utility/vk_format_utils.h

```diff
@@ -270,9 +270,6 @@ inline VkExtent3D vkuFormatTexelBlockExtent(VkFormat format);
 // Returns the Compatibility Class of a VkFormat as defined by the spec
 inline enum VKU_FORMAT_COMPATIBILITY_CLASS vkuFormatCompatibilityClass(VkFormat format);
 
-// Return true if a VkFormat is 'normal', with one texel per format element
```

Comment:
`normal` here is from what I can tell just a bad comment left over from years of trying to get the original format checks correct in VVL